### ISSUE TITLE
Implement normalization 

### ISSUE THREAD

**Issue by cells2numbers** [cytomining/cytotools#11]
The cytominerScript function normalize.R (https://github.com/broadinstitute/cytominer_scripts/blob/master/normalize.R) is missing and needs to be implemented in cytotools.

**Comment by mcquin** [comment]
We'll drop support for single-cell normalization.

We're unsure how to resolve single-cell column names with the mutated column names of an aggregated population (e.g., columns ending in `_median`).

Let's rethink our data structures such that they:
- are tidy
- reflect the transformations applied*

\* Though, if there's a way to encode this information outside of the data (e.g., column names and table/csv names), we should explore that.

**Comment by mcquin** [comment]
```sh
./normalize.R \
    -b batch0   \
    -p SQ00015116 \
    -s "Metadata_broad_sample_type == '''control'''"
```
expected output: backend/batch0/SQ00015116/SQ00015116_normalized.csv

---
```sh
./normalize.R \
    -b batch0 \
    -p SQ00015116 \
    -c \
    -s "Metadata_broad_sample_type == '''control'''"
```
expected output: backend/batch0/SQ00015116/SQ00015116_normalized_single_cell.csv

**Comment by shntnu** [comment]
> * Though, if there's a way to encode this information outside of the data (e.g., column names and table/csv names), we should explore that.

Single cell support is preserved

We should encode the information elsewhere,  at the very least through code in reproducible workflows

**Comment by shntnu** [comment]
Addressed in #12
